etc/kubernetes/manifests/overlays/local/keycloak/route.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: route.openshift.io/v1
+kind: Route
```

Comment:
the local overlay is meant to be deployed on plain kubernetes, so I guess this should be part of the qe overlay